docs/modules/ROOT/pages/guide-upgrades.adoc

```diff
@@ -34,14 +39,16 @@ To get started, you’ll need the following:
 
 1. *A Defender account*. Head over to https://defender.openzeppelin.com[Defender] to sign up for a new account. 
 2. *ETH to pay for transactions gas*. Transactions require gas for execution, so make sure to have some ETH available. For this guide we will use Rinkeby ETH.
-3.  *Multi Sig*. A multisig contract to control our upgradeable contract. In this guide we will use a Gnosis Safe but you could also use any supported multisig such as a legacy Gnosis multisig. https://help.gnosis-safe.io/en/articles/3876461-create-a-safe-multisig[Create a Gnosis Safe] multisig on the Rinkeby network, with M > N/2 and M >1. This should be at least 2 of 3. (See xref:advisor.adoc[Advisor] for guidance on multisig best practices).
-4.  *Hardhat project*. A xref:learn::developing-smart-contracts.adoc#setting-up-a-solidity-project[Hardhat project] with xref:upgrades-plugins::hardhat-upgrades.adoc[Hardhat Upgrades plugin], Hardhat Defender and ethers.js installed. 
+3.  *Multi Sig*. A multisig contract to control our upgradeable contract. In this guide we will use a [Gnosis Safe](https://safe.gnosis.io/) but you could also use any supported multisig such as a legacy [Gnosis MultiSigWallet](https://github.com/gnosis/MultiSigWallet). 
```

Comment:
I think we are using markdown instead of asciidoc here (and I think it was my getting it wrong in the suggestion)

Reply:
Good catch.  Sorry I didn't pick this up.